### ISSUE TITLE
Body param array with user defined struct produces bad output

### ISSUE THREAD

**Issue by Kelwing** [swaggo/swag#547]
**Describe the bug**
When I specify a parameter as a slice of a custom struct in the body, it produces invalid output.  The schema type is set to the name of the struct instead of 'array'.

**To Reproduce**
Specify a Param as follows
```
@Param mystruct body []MyStruct true "MyStruct"
```

It produces the following schema:
```
        schema:
          items:
            $ref: '#/definitions/main.MyStruct'
          type: MyStruct
```

**Expected behavior**
Schema should be:
```
        schema:
          items:
            $ref: '#/definitions/main.MyStruct'
          type: array
```

**Your swag version**
v1.6.3

**Your go version**
1.12.3

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Chrome
 - Version: 77.0.3865.120

**Additional context**
N/A

**Comment by gmarcial** [comment]
Ya

I find myself in the same problem...

**Comment by sdghchj** [comment]
fixed by PR #579
